test/core/transport/chttp2/settings_timeout_test.cc

```diff
@@ -179,11 +179,11 @@ class Client {
       gpr_log(GPR_INFO, "OnEventDone(): %s", grpc_error_string(error));
```

Comment:
It is unclear to me if OnEventDone is expected to only be called when mu_ (see PollUntilDone) is held (and therefore not doing any synchronization here is intentional). If so, there's a problem somewhere deeper in C-core stack.
If not synchronizing access to `done_` properly is just an omission, the fix is simple (= the diff in this PR).